packages/ui/src/styles/globals.css

```
@@ -271,6 +274,35 @@
+  animation-timing-function: linear;
+  animation-fill-mode: both;
+  animation-timeline: scroll(root block);
+  animation-range: calc(100% - 180px) calc(100% - 20px);
+
+  /* Named only where scroll timelines exist. Elsewhere `animation-timeline` is
+     dropped, so the animation would run once on the document timeline and
+     `fill-mode: both` would hold the bar off screen for good. Unlike
+     `mark-frame` below, these keyframes have a `to`, so there is no safe
+     settled state to fall back on — better to not animate at all. */
+  @supports (animation-timeline: scroll()) {
+    animation-name: nav-exit;
```

Stylelint (17.14.0)

[error] Invalid position for declaration (no-invalid-position-declaration)

(no-invalid-position-declaration)